packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -170,7 +183,7 @@ export const ChatList = ({
                   chat={c}
                   myId={chatListState.userId}
                   isSelected={c.id === selectedChatId}
-                  isRead={c.id === selectedChatId}
+                  isRead={c.id === selectedChatId || (c.isRead ?? false)}
```

Comment:
probably makes more sense to do this...

`isRead={c.id === selectedChatId || c.isRead}`

c.isRead is a boolean so it cannot be null or undefined; or if it can then you have a type declaration issue.